azext_iot/digitaltwins/params.py

```diff
@@ -242,20 +279,64 @@ def load_digitaltwins_arguments(self, _):
         context.argument(
             "servicebus_resource_group",
             options_list=["--servicebus-resource-group", "--sbg"],
-            help="Name of ServiceBus resource group.",
+            help="Name of ServiceBus resource group. If not provided, the Digital Twin instance "
+            "resource group will be used.",
             arg_group="Service Bus Topic",
         )
         context.argument(
             "endpoint_subscription",
             options_list=["--servicebus-subscription", "--sbs"],
-            help="Name or ID of subscription where the endpoint resource exists. "
+            help="Name or id of subscription where the endpoint resource exists. "
             "If no subscription is provided the default subscription is used.",
             arg_group="Service Bus Topic",
         )
 
     with self.argument_context("dt endpoint wait") as context:
         context.ignore("updated")
 
+    with self.argument_context("dt identity assign") as context:
+        context.argument(
+            'system_identity',
+            options_list=['--mi-system-assigned', '--system'],
+            arg_type=get_three_state_flag(),
+            help="Assign a system-assigned managed identity to this Digital Twin instance."
```

Comment:
Digital Twin should be plural "...to this Digital Twin**s** instance."

Reply:
will make an extra pr with these fixes